### DISCUSSION TITLE
v3.0.0-rc.6

### DISCUSSION THREAD

**Discussion by pi0** [nuxt/nuxt#18388] [Announcements]
✅ RC.6 is a hotfix release fixing known issues with previous releases

[Release Notes](https://github.com/nuxt/framework/releases/tag/v3.0.0-rc.6)

**Comment by FileSoft-us** [comment]
Thanks...

**Comment by mamlzy** [comment]
Thank you! can't wait for the stable release..😵🙌

**Reply by FrancoLab** [reply]
Anyone knows when this will happen?

**Reply by Harm-Nullix** [reply]
Summer!

**Comment by rahulkumarsingh73690** [comment]
Waiting for the stable release 😍

**Reply by OrionQuantum342244** [reply]
We are all waiting

**Comment by r-this** [comment]
Exciting, thanks!

**Comment by benoit-outsourcify** [comment]
We've started an app aimed to be release in prod in October 2022 months and we chose Nuxt 3 instead of Nuxt 2 (which we have been using for years).
Hopefully the stable release comes very soon, we were hoping for June 2022 as that's what was estimated.

**Reply by Harm-Nullix** [reply]
Nice! You are on the same path as me my friend!

**Comment by Dorianslavov** [comment]
Since I've upgraded to RC 6, my application's speed has gone down drastically in terms of **Page loading** / **HMR** / **Dev server load time**... Anyone experiencing this too ?

**Comment by FrancoLab** [comment]
Any update on when the stable release will come out? Summer is almost over...

**Reply by FrancoLab** [reply]
I cant, and I am so scared to build out a whole app with Nuxt3 and they take another 3 months to release the stable version

**Reply by rahulkumarsingh73690** [reply]
They promised in June but till now no confirmation 🙄

**Reply by FrancoLab** [reply]
@pi0 would you mind giving us an update?

**Reply by rahulkumarsingh73690** [reply]
@Atinux @antfu @danielroe

**Reply by danielroe** [reply]
The latest is http://v3.nuxtjs.org/community/roadmap. Summer.

**Reply by rahulkumarsingh73690** [reply]
@danielroe Please tell us the expected date not summer

**Reply by FrancoLab** [reply]
@danielroe summer in Africa? Summer in Europe? I think the community deserves a date

**Reply by FrancoLab** [reply]
@Atinux

**Reply by pi0** [reply]
@FrancoLab @rahulkumarsingh73690. I can understand your concerns regarding the delay and that you expected it earlier and on a specified date.

We initially planned to announce a 3.0.0 on 2nd June 2022 but then decided to delay this to ensure when we tag 3.0.0 both Vue and Nuxt ecosystem is also stable. As of today RC builds of Nuxt 3 are stable enough that many already started to deploy their production apps with Nuxt 3 and all changes are tracked in release notes just like normal release cycles.

Summer 2022 ends by **September 23** and we aim to announce 3.0.0 **before this date**.

**Reply by FrancoLab** [reply]
Thank you @pi0 for the response. This gives us great clarity. Appreciate the response.

**Reply by targetlucked69** [reply]
Damn, I hate devs like you @rahulkumarsingh73690 . If you want a faster release then go contribute. You didn't even say thanks after a good response. Demanding af

**Reply by rahulkumarsingh73690** [reply]
@walmartwarlord 

Sorry I don't have time to contribute and forget to say thanks 

I am waiting for nuxt3 from previous year expecting release in June 

@pi0 Thanks for information, started rewriting app in nuxt 3 rc 6

**Reply by akshay-sood** [reply]
@pi0 This is beautiful news! I am also awaiting for nuxt3 prod eagerly, and was sad not finding any definite release information after June passed. I really love nuxt and did not want to switch to next. This clears the air! Thanks for the response, and thanks a ton to the team for such a wonderful framework making our lives as devs a lot easier!

**Reply by chstappert** [reply]
Yes, true @akshay-sood. A big thanks to the whole nuxt team. There is a lot of progress, just have a look into the repo. ;-) We'll start with rc6 and 3.0.0 will "just" an update cycle. Thanks for the great news!
